test/classes/nilability/nilability-upcasting.chpl

```diff
@@ -128,6 +128,8 @@ module test {
         writeln(x.type:string, " ", x);
         var y = cb:borrowed;
         writeln(y.type:string, " ", y);
+        var z = cb:borrowed class;
+        writeln(y.type:string, " ", y);
```

Comment:
Please `z` and not `y` :)

I checked the other cases you added in this test - they look correct in this regard.

Reply:
Thanks for catching that!